buildSrc/src/main/java/org/elasticsearch/gradle/ElasticsearchDistribution.java

```diff
@@ -197,15 +197,15 @@ public Configuration getConfiguration() {
     void finalizeValues() {
 
         if (getType() == Type.INTEG_TEST_ZIP) {
-            if (platform.isPresent()) {
+            if (platform.getOrNull() != null) {
```

Comment:
I'm confused, does `isPresent()` return `true` for a `null` value?

Reply:
It does, as long as a value is explicitly set to null.